packages/react-icons/src/components/OneDriveIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import createSvgIcon from '../utils/createSvgIcon';
+
+const OneDriveIcon = createSvgIcon({
+  svg: ({ classes }) => (
```

Comment:
Minor suggestion/thought:

will there ever be more than 1 class name here?  Maybe it should just be className.

Reply:
In our experience with react-icons-northstar there are cases like this, so would rather leave it as is at this moment as it is easier for extending this way and won't introduce necessity for changes in all icons in the future..